.travis/run_travis_job.sh

```diff
@@ -47,7 +47,7 @@ then
     echo 'Rust checks are completed.'
 else
     cd "${TRAVIS_BUILD_DIR}"
-
+    cd exonum-java-binding-parent
```

Comment:
I expect that for new modules we'll create new stages in the Travis build matrix + modify this sh script accordingly.